nixos/modules/services/monitoring/osquery/default.nix

```diff
@@ -0,0 +1,78 @@
+{ config, lib, pkgs, ... }:
+
+with builtins;
+with lib;
+let
+  cfg = config.services.osquery;
+  flags = import ./flags.nix { inherit cfg lib pkgs; };
+in
+{
+  options.services.osquery = {
+    enable = mkEnableOption (mdDoc "osqueryd daemon");
+
+    settings = mkOption {
+      default = { };
+      description = mdDoc ''
+        Configuration to be written to the osqueryd JSON configuration file.
+        To understand the configuration format, refer to https://osquery.readthedocs.io/en/stable/deployment/configuration/#configuration-components.
+      '';
+      example = {
+        options.utc = false;
+      };
+      type = types.attrs;
```

Comment:
can be `attrs` right?

Reply:
I'm not sure I understand. I don't have a `with types` that brings `attrs` into scope. Is `with types; attrs` preferred over `types.attrs`?

Reply:
I don't see a reason to change this tbh.

Reply:
Sorry for the comment, I thought `types` was in the context.